.gitignore

```
@@ -1,47 +1,58 @@
-# OS files
-.DS_Store
-*.orig
-*.bak
-*~
-*.tmp
-*.rej
-\#*\#
-*.sublime-workspace
-
-# Xcode
-xcuserdata/
-build
-CoreAudioUtilityClasses
-CoreAudioUtilityClasses.zip
-
-# Unity
-Library
-bin
-obj
-UserSettings/
-Temp/
+# Local files and folders generated by Unity
+[Bb]uild/
+[Bb]uilds/
+[Bb]uild/
+[Bb]uilds/
+CodeCoverage/
+Library/
 Logs/
-*.orig.*
-
-# VS
-*.csproj.user
-!PlatformDependent/WinRT/Tests/VSUnitTests/Assembly-CSharp/Assembly-CSharp.csproj.user
+obj/
+Temp/
+UserSettings/
+Assets/EntityCache
+Assets/EntityCache.meta
+Assets/InitTestScene*
+Assets/GlobalEntitiesDependencies/
+Assets/GlobalEntitiesDependencies.meta
+Assets/SceneDependencyCache/
+Assets/SceneDependencyCache.meta
+Assets/StreamingAssets/
+Assets/StreamingAssets.meta
+Assets/Resources/PerformanceTestRunInfo.json
+Assets/Resources/PerformanceTestRunInfo.json.meta
+
+# IDE project files
+*.csproj
+*.sln
 *.suo
-*.vcxproj.user
-.vs/
-
-# VS intellisense files next to solution file for C++ projects - global configuration should perhaps place it elsewhere
-*.sdf
-
-# temporary locking file for solution access
-*.opensdf
+.idea
+.vs
+.vscode
+.vsconfig
 
-# resharper
-*.sln.DotSettings.user
+# ReSharper user settings file
+*.DotSettings.user
 
-# MonoDevelop
-*.userprefs
-
-# ctags:
-.tags
-.tags_sorted_by_file
+# Files generated by merging and stashing
+*.orig
+*.orig.meta
+*.rej
+*.rej.meta
+*_BASE_*
+*_LOCAL_*
+*_REMOTE_*
+*~Stashed changes
+
+# Files generated by code formatting tools
+*.uncrustify
+*.uncrustify.*
+*.preformat.bak*
+
+# Files and Folders generated when running upm-ci locally
+.bin
+.download
+.Editor/
+upm-ci~/
+
+# Overrides
+!*~
```

```suggestion
!*~
SDK/VisualStudio/Debug
SDK/VisualStudio/x64
```

These are build artifacts and should not be committed as they are in this PR.

---

Not an expert in XCode but looks like a bunch of user settings and build artifacts were committed as well.

---

Yep, we need to fix the .gitignore file.

---

Aye, more copy-pasta.